src/libraries/System.ComponentModel.TypeConverter/tests/default.rd.xml

```diff
@@ -0,0 +1,15 @@
+<Directives>
+  <Application>
+    <Assembly Name="mscorlib" />
```

Comment:
I'm a little bit worried that such broadly applied RD.XML will hide product issues. But also I don't know how much of TypeConverter actually is compatible with trimming and whether it even makes sense to guard trim safety.

Just thought to write down what I think.

Reply:
I disabled 7 test scenarios, all in `XTypeDescriptionProviderTests`, to get rid of the rd.xml dependency.